src/server/database/migrations/20160624120200-update-user-profile.js

```diff
@@ -0,0 +1,14 @@
+exports.up = async r => {
+  return await r.table('UserProfile').update({
+    isNew: false,
+    welcomeSentAt: null,
+    preferredName: '',
+    memberships: []
```

Comment:
Memberships isn't part of the database, that's just a convenience in graphql to link to the teammember table.